src/providers/snackbars/snackbar.tsx

```diff
@@ -65,4 +65,5 @@ const SnackbarsContainer = styled.div`
   max-width: 360px;
   width: 100%;
   display: grid;
+  z-index: ${zIndex.overlay};
```

Comment:
Let's actually use some higher value, so that snackbars are on top of the video workspace overlay when visible. Maybe `nearSheetOverlay`?